ajax/libs/jsonld/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "jsonld",
-  "version": "0.4.12",
+  "version": "1.2.1",
```

Comment:
I can merge this once the version change is reverted, as the new version hasn't been added yet. Would you? Thanks.